peer/brontide.go

```diff
@@ -1422,6 +1431,13 @@ out:
 
 			discStream.AddMsg(msg)
 
+		case *lnwire.Custom:
+			err := p.cfg.HandleCustomMessage(p.PubKey(), msg)
```

Comment:
If there's no custom message handler, then we should treat this as an error. In addition, if the message's type is even, even if there's a handler, we should likely return an error state.

Reply:
Added error case, even though a handler is always registered currently.

Not sure about even types. `lnd` doesn't know whether the external handler understands an even message? This would probably require an ack from the external handler, similar to what the htlc interceptor does. But then also with support for multiple handlers. I think that would make this PR a whole lot more complex, because the messages stay up in the air until the external process has given a response. And if I am seeing this right, the only difference is in what is logged locally? Because there isn't an actual response message on the wire.